packages/components/src/public/QueryColumn.ts

```diff
@@ -433,7 +433,7 @@ export class QueryColumn implements IQueryColumn {
         const lcName = importName.toLowerCase().trim();
         return (
             this.caption?.toLowerCase() === lcName ||
-            this.caption?.replace(' ', '').toLowerCase() === lcName ||
+            this.caption?.replaceAll(' ', '').toLowerCase() === lcName || // Issue 52193: allow for "SampleID" when caption is "Sample ID", but don't match for simply fewer spaces
```

Comment:
we don't have to make this change for this PR, but after looking at this, do you think this method should be renamed? Seems less about `isImportColumn` and more like `isColumnIdentifierMatch` or something along those lines.

Reply:
Yeah, just couldn't come up with a good name ...

Reply:
renames. Hope it makes it more clear.